### PR TITLE
build: provide jest integration

### PR DESCRIPTION
**Pull request by feedm3**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,21 +38,28 @@
     "styled-components": "^5.0.1"
   },
   "devDependencies": {
-    "@types/node": "13.9.8",
-    "@types/react": "16.9.30",
-    "@types/react-dom": "16.9.5",
+    "@testing-library/jest-dom": "^5.3.0",
+    "@testing-library/react": "^10.0.2",
```
Comment (georgiee) [inline package.json:42]: Testing Library for a win 🤘

**Review by georgiee — APPROVED** [review]
Works, runs locally, code changes look good too 👌

**Comment by georgiee** [comment]
Please provide at least a small description of your PRs and what would be the best verification approach like "Run the test locally and wait for the first test to succeed". This will help your reviewers a lot.

**Comment by georgiee** [comment]
Also your commit message "feat: docs" & "feat: setup jest" are not the best.
I would suggest you squash & merge and to use something like `build: provide jest integration` if possible.

**Comment by feedm3** [comment]
> Please provide at least a small description of your PRs and what would be the best verification approach like "Run the test locally and wait for the first test to succeed". This will help your reviewers a lot.

Yep, you are right. Will do with the next PRs 💪

**Pull request merged**